class/defaults.yml

```diff
@@ -1,3 +1,7 @@
 parameters:
   openshift4_version:
-    namespace: syn-openshift4-version
+    namespace: syn
+    spec:
+      channel: stable-4.4
+      clusterID: "${openshift:clusterID}"
```

Comment:
Unfortunately this will break compiles for clusters where `openshift.clusterID` isn't defined. Until https://github.com/projectsyn/commodore/issues/138 is fixed, we can't use variables in `defaults.yml`.